tasks/main.yml

```diff
@@ -9,24 +9,24 @@
 
 - name: Download and install Terraform binary
   ansible.builtin.unarchive:
-    creates: "{{ install_dir }}/terraform"
-    dest: "{{ install_dir }}"
-    group: "{{ group }}"
-    mode: "{{ mode }}"
-    owner: "{{ owner }}"
+    creates: "{{ terraform_install_dir }}/terraform"
+    dest: "{{ terraform_install_dir }}"
+    group: "{{ terraform_group }}"
+    mode: "{{ terraform_mode }}"
+    owner: "{{ terraform_owner }}"
     remote_src: yes
     src: https://releases.hashicorp.com/terraform/{{ terraform_version }}/terraform_{{ terraform_version }}_linux_amd64.zip
 
 - name: Download and install terraform-docs binary
   ansible.builtin.unarchive:
-    creates: "{{ install_dir }}/terraform-docs"
-    dest: "{{ install_dir }}"
-    group: "{{ group }}"
+    creates: "{{ terraform_install_dir }}/terraform-docs"
+    dest: "{{ terraform_install_dir }}"
+    group: "{{ terraform_group }}"
     # The archive also includes a README and a LICENSE file, but we
     # don't want those.
     include: terraform-docs
-    mode: "{{ mode }}"
-    owner: "{{ owner }}"
+    mode: "{{ terraform_mode }}"
+    owner: "{{ terraform_owner }}"
     remote_src: yes
     # ansible-lint generates a warning that "lines should be no longer
     # than 160 characters" here", but there is no good way to break up
```

Comment:
Can we change the `noqa 204` below to use the rule name instead?

Reply:
@jsf9k I removed the `noqa` code in order to get ansible-lint to hopefully spit out the rule name, and ansible-lint didn't complain at all....should I just remove the ignore directive?

Reply:
Yes, please do that.

Reply:
Done in [69271e1](https://github.com/cisagov/ansible-role-terraform/pull/24/commits/69271e160e2a1d050e1ba246615cdf27f557f16f)